### PR TITLE
Remove 'Deploy to Openshift' command

### LINKED ISSUE #605: Deploy to OpenShift fails with 'command `openshift.component.deployRootWorkspaceFolder` not found'
It seems like the command was renamed to `openshift.component.createFromRootWorkspaceFolder`.

### LINKED ISSUE #609: Integrate vscode-extension-proposals
see https://github.com/redhat-developer/vscode-extension-proposals

possible recommendations:
- ~~deploy to openshift~~ or adding the openshift quarkus extension --> install OpenShift Tools
- launching debug when there is a `devfile.yaml` present in the workspace folder --> install OpenShift Tools

### PR DESCRIPTION
**Pull request by datho7561**
- Remove the 'Deploy to OpenShift' command, since it doesn't work and isn't being used.
- Replace our existing system to recommend OpenShift Toolkit with vscode-extension-proposals

Fixes https://github.com/redhat-developer/vscode-quarkus/issues/605, Closes https://github.com/redhat-developer/vscode-quarkus/issues/609

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

**Comment by datho7561** [comment]
Actually, I'm going to integrate the extension recommender into this. One second...

**Comment by datho7561** [comment]
Okay. Should be good for review.

File: src/utils/recommendationUtils.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+let recommendationService: IRecommendationService;
+
+/**
+ * Initialize the recommendation service
+ *
+ * @param context the extension context
+ */
+export function initRecommendationService(context: ExtensionContext) {
+  if (!recommendationService) {
+    recommendationService = RecommendationCore.getService(context);
```
Comment (rgrunber) [inline src/utils/recommendationUtils.ts]: If you look at the API, you'll see that `getService` takes an optional telemetry service. If you provide the telemetry service that was initialize just above where this method is called, the results of the recommendation will be recorded.

**Review by rgrunber — COMMENTED** [review]
Looks good overall, just one thing I noticed.

**Pull request merged**